kudo-operator/docs/latest/configuration.md

```diff
@@ -1,2 +1,77 @@
 KUDO Spark Operator Configuration
 ---
+
+## Default configuration
+The default KUDO Spark configuration enables installation of the following resources:
+* CRDs (Custom Resource Definitions) for `SparkApplication` and `ScheduledSparkApplication`
+* Service Account, `ClusterRole`, and `ClusterRoleBinding` for Operator
+* Service Account, `Role`, and `RoleBinding` for Spark Applications (Drivers)
+* Spark Operator Controller which handles submissions and launches Spark Applications
+* Mutating Admission Webhook for customizing Spark driver and executor pods based on the specification
+* `MetricsService` for Spark Operator and Spark Applications to enable metrics scraping by Prometheus
+
+Full list of configuration parameters and defaults is available in KUDO Spark [params.yaml](../../operator/params.yaml).
+## Docker
+Docker images used by KUDO Spark and image pull policy can be specified by providing the following parameters:
+```bash
+kubectl kudo install spark --instance=spark-operator \
+        -p operatorImageName=mesosphere/kudo-spark-operator \
+        -p operatorVersion=latest \
+        -p imagePullPolicy=Always
+```
+
+## Monitoring
+Metrics reporting is enabled by default and can be disabled by providing `-p enableMetrics=false` parameter:
+```bash
+kubectl kudo install spark --instance=spark-operator -p enableMetrics=false
+```
+When enabled, KUDO Spark installs Metrics Services and Service Monitors to enable metrics scraping by Prometheus. More
+information about how KUDO Spark exposes metrics and metrics configuration  is available in [KUDO Spark Operator Monitoring](./monitoring.md)
+documentation.
+
+## Spark History Server
+KUDO Spark comes bundled with Spark History Server which is disabled by default. It can be enabled via `-p enableHistoryServer=true`:
+```bash
+kubectl kudo install spark --instance=spark-operator -p enableHistoryServer=true
+```
+
+More information about Spark History Server configuration is available in [Spark History Server Configuration](./history-server.md)
+documentation.
+
+## Service Accounts and RBAC
+By default, KUDO Spark will create service accounts, roles, and role bindings with preconfigured permissions required
```

Comment:
```suggestion
By default, KUDO Spark will create service accounts, roles, and role bindings with pre-configured permissions required
```

Reply:
tha original version [seems to be valid](https://www.merriam-webster.com/dictionary/preconfigure)